### PR TITLE
#34 add equation

### PR DESCRIPTION
**Pull request by Wiki13002**
-equation display
-enter the result of the equation

### CODE REVIEW

**Comment by ani119** [comment]
@Wiki13002 please reslove conflicts

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Script : MonoBehaviour
+{
+    int x;
```
Comment (ani119) [inline podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs:8]: We use English name of variables and functions, please look to ours code and stay in the same pattern.

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    int y;
+    string rownanie;
+    public Text napisRownanie;
+    public InputField odpowiedz;
```
Comment (ani119) [inline podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs]: public variables should be upper than private ones

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.KeypadEnter))
+        {
+            sprawdzOdpowiedz();
+        }
+    }
+    public void LosujRownanie()
+    {
+        x = Random.Range(2, 11);
```
Comment (ani119) [inline podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs]: We exclude 1*2 ? And why there is (2,11)?

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        if (odpowiedz.text == (x * y).ToString())
+        {
+            blad.text = "Bardzo dobrze następne równanie";
+            LosujRownanie();
+
+        }
+        else
+        {
+            blad.text = "spróbuj jeszcze raz";
```
Comment (ani119) [inline podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs]: Let's start it with big letter

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Script : MonoBehaviour
+{
+    int x;
+    int y;
+    string rownanie;
+    public Text napisRownanie;
```
Comment (ani119) [inline podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs]: We use English name of variables and functions, please look to ours code and stay in the same pattern.

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs [hunk 6]
```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Script : MonoBehaviour
```
Comment (ani119) [inline podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs]: Please name this script with count what it do (ofc in English)

**Review by ani119 — CHANGES_REQUESTED** [review]
As I wrote in single comments this code needs some work, pleace use pattern which all team use - english names of variables and functions (remember about names gameobjects in scene hierarchy too). Your script should have name which suggest what the script do. Some refactoring about the variable order, as I wrote a few times the public ones should be higher. More comments are written at specific lines. Please remeber to be up to date with your branch and don't have conflicts.

File: podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs (hunk 4 above)
Comment (przemytka) [reply podwodny_swiat_tabliczki_mnozenia/Assets/Scritps/Equation.cs]: Capital letter* :P

**Review by ani119 — DISMISSED** [review]
Good job!

**Review by przemytka — DISMISSED** [review]

**Review by AgaMazur — CHANGES_REQUESTED** [review]
Delete unnecessary empty lines in 'multiplication' code

**Review by ani119 — APPROVED** [review]

**Review by przemytka — APPROVED** [review]

**Review by juliakobusinska98 — APPROVED** [review]

**Pull request merged**